### ISSUE TITLE
Support sites with multiple feeds

### ISSUE THREAD

**Issue by Zetagon** [gugray/rss-parrot#34]
It seems like rss-parrot creates one account per website, but that fails when a website has multiple rss feeds.

To reproduce
Message the bot with example.com/feed1.xml.  An account should be created for example.com.

Message the bot with example.com/feed2.xml.  You should get a reply with the previously created account which has the wrong feed.

**Comment by gugray** [comment]
This is a duplicate of #11. Unfortunately for now this is a limitation: RSS Parrot only supports one URL per feed (or one feed per URL, whichever way you want to look at it).

Note that you can have different feeds per different paths, i.e. this works:
example.com/dogs/feed.xml
example.com/cats/feed.xml

**Comment by jordemort** [comment]
I wasn't able to get multiple feeds from here working: https://github.com/eikowagenknecht/lootscraper

I wanted both the Amazon and the Epic Games feed, but after creating the Amazon one, trying to create the Epic one just points me at the same user that was created for Amazon.

**Comment by gugray** [comment]
Yes, unfortunately for now multiple feeds from the same URL won't work. They will work if the the link in the different feeds designates differne URLs: `<link href="https://phenx.de/loot" rel="alternate"/>`

E.g., https://phenx.de/loot/ubisoft_game, https://phenx.de/loot/amazon_dame etc.

**Comment by chri2** [comment]
Hey, thanks for the great service! Ended up here, because I ran into the same problem:

https://www.goettinger-tageblatt.de/arc/outboundfeeds/rss/category/der-norden/
https://www.goettinger-tageblatt.de/arc/outboundfeeds/rss/category/lokales/goettingen-lk/goettingen/

I ended up configuring a nginx reverse proxy to have a different site in the URL and rewrite the URLs inside the feed.

**Comment by bazzargh** [comment]
I noticed another instance of this, where there's _not_ multiple feeds per domain but the old one went away! In this case it's www.electoral-vote.com . I'd noticed the feed had gone silent in January, finally went to check and found the feed is listed in the html

```
<link rel="alternate" type="application/rss+xml" title="RSS" href="/evp2026/RSS/index.rss">
```

... so the problem is obvious, they switched at the end of the year. Sure enough, checking /evp2025/RSS/index.rss last year's feed is there.

I tried asking the birb for `http://www.electoral-vote.com/evp2025/RSS/index.rss` directly but I take it you're saying the  `<link>https://www.electoral-vote.com/</link>` in the rss (before and after) means the new one can't replace the old one?
